.github/workflows/tests-nightly.yml

```
@@ -212,16 +212,18 @@ jobs:
 
       - name: Set tag for tests
         id: set-tag-for-tests
+        uses: actions/github-script@v6
         env:
           is_old_version: ${{ needs.get-environment-variables.outputs.is_old_version }}
-          old_version_test_tag: "@oldRelease"
-          new_version_test_tag: "@stable"
-        run: |
-          if [[ $is_old_version == "true" ]]; then
-              echo "TEST_TAG=$old_version_test_tag" >> $GITHUB_OUTPUT
-          else
-              echo "TEST_TAG=$new_version_test_tag" >> $GITHUB_OUTPUT
-          fi
+        with:
+          result-encoding: string
+          script: |
+            const { is_old_version } = process.env
```

Shouldn't it be this?
```suggestion
            const { is_old_version } = process.env.is_old_version
```